librz/core/core.c

```diff
@@ -2446,3 +2446,64 @@ RZ_API RzCmdStatus rz_core_core_plugins_print(RzCore *core, RzCmdStateOutput *st
 	rz_cmd_state_output_array_end(state);
 	return RZ_CMD_STATUS_OK;
 }
+
+/**
+ * \brief Executes a command and returns the filtered output.
+ *
+ * \param core Pointer to RzCore instance.
+ * \param command Command to be executed.
+ * \param filter String used to filter the output.
+ * \return Filtered output of the command. Must be freed by the caller.
+ */
+RZ_API char *rz_core_filter_command_output(RZ_NONNULL RzCore *core, const char *command, const char *filter) {
```

Comment:
Please add ownership mark to the return value. Also, instead of filtering command output, just accept any string. And instead of calling `afb`, in the corresponding place in visual.c just call function that would return the same as `afb` and filter the output using this function.